plone/app/z3cform/templates/text_input.pt

```diff
@@ -0,0 +1,37 @@
+<html xmlns="http://www.w3.org/1999/xhtml"
```

Comment:
IMO all widget templates should be defined in z3c.form only - in the most minimal way possible. Modernizing for HTML5 compatibility should be enough.
I dislike weaving bootsrap or any other CSS framework specific markup or classes directly into core templates. That creates a hard dependency which is not easy to get rid of.
Of course we also need CSS classes but those should be set BEM style and any customizations be done in CSS only.